From 2d1d2164421a36789a5bc40977eea31762981dca Mon Sep 17 00:00:00 2001
From: Stefan Lippers-Hollmann <s.l-h@gmx.de>
Date: Mon, 17 Mar 2014 06:29:47 +0100
Subject: [PATCH 362/390] rtl8192du: declare MODULE_FIRMWARE() macros

Declare the firmware images used by rtl8192du, which helps to map
modules and required firmware images.

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
---
 hal/rtl8192d_hal_init.c | 5 +++++
 1 file changed, 5 insertions(+)

--- a/drivers/staging/rtl8192du/hal/rtl8192d_hal_init.c
+++ b/drivers/staging/rtl8192du/hal/rtl8192d_hal_init.c
@@ -308,6 +308,11 @@ exit:
 	release_firmware(fw);
 	return rtstatus;
 }
+#ifdef CONFIG_WOWLAN
+MODULE_FIRMWARE("rtlwifi/rtl8192dufw_wol.bin");
+#else
+MODULE_FIRMWARE("rtlwifi/rtl8192dufw.bin");
+#endif /* CONFIG_WOWLAN */
 
 /*	Description: Download 8192D firmware code. */
 int FirmwareDownload92D(struct rtw_adapter *adapter, bool bUsedWoWLANFw)
